Include/internal/pycore_gc.h

```diff
@@ -382,6 +382,9 @@ extern void _Py_ScheduleGC(PyThreadState *tstate);
 extern void _Py_RunGC(PyThreadState *tstate);
 
 
+// Functions to clear generator frames
+extern int _PyGC_VisitFrameStack(struct _PyInterpreterFrame *frame, visitproc visit, void *arg);
```

Comment:
This doesn't clears generator frames right, maybe change comment that it just visits it?